pkg/api/server/register_images.go

```diff
@@ -10,32 +10,652 @@ import (
+	// - application/json
+	// responses:
+	//   '200':
+	//     schema:
+	//     items:
+	//       "$ref": "TBD"
+	//     description: no error
+	//   '500':
+	//     description: unexpected error
+	//     schema:
+	//      "$ref": "#/types/ErrorModel"
 	r.Handle(VersionedPath("/images/{name:..*}/get"), APIHandler(s.Context, generic.ExportImage)).Methods(http.MethodGet)
+	// swagger:operation GET /images/{nameOrID}/history images createImage
+	//
+	// History of an image
+	//
+	// ---
+	// parameters:
+	//  - in: path
+	//    name: nameOrID
+	//    required: true
+	//    description: the name or ID of the container
+	// produces:
+	// - application/json
+	// responses:
+	//   '200':
+	//     schema:
+	//     items:
+	//       "$ref": "#/types/HistoryResponse"
+	//   '404':
+	//     description: no such image
+	//     schema:
+	//       "$ref": "#/types/ErrorModel"
+	//   '500':
+	//     description: unexpected error
+	//     schema:
+	//      "$ref": "#/types/ErrorModel"
+	r.Handle(VersionedPath("/images/{name:..*}/history"), APIHandler(s.Context, handlers.HistoryImage)).Methods(http.MethodGet)
+	// swagger:operation GET /images/{nameOrID}/json images inspectImage
+	//
+	// Inspect an image
+	//
+	// ---
+	// parameters:
+	//  - in: path
+	//    name: nameOrID
+	//    required: true
+	//    description: the name or ID of the container
+	// produces:
+	// - application/json
+	// responses:
+	//   '200':
+	//     schema:
+	//     items:
+	//       "$ref": "#/types/imageInspect"
+	//   '404':
+	//     description: no such image
+	//     schema:
+	//       "$ref": "#/types/ErrorModel"
+	//   '500':
+	//     description: unexpected error
+	//     schema:
+	//      "$ref": "#/types/ErrorModel"
 	r.Handle(VersionedPath("/images/{name:..*}/json"), APIHandler(s.Context, generic.GetImage))
+	// swagger:operation POST /images/{nameOrID}/tag images tagImage
+	//
+	// Tag an image
+	//
+	// ---
+	// parameters:
+	//  - in: path
+	//    name: nameOrID
+	//    required: true
+	//    description: the name or ID of the container
+	//  - in: query
+	//    name: repo
+	//    type: string
+	//    description: the repository to tag in
+	//  - in: query
+	//    name: tag
+	//    type: string
+	//    description: the name of the new tag
+	// produces:
+	// - application/json
+	// responses:
+	//   '201':
+	//     description: no error
+	//   '400':
+	//     description: bad parameter
+	//     schema:
+	//       "$ref": "#/types/ErrorModel"
+	//   '404':
+	//     description: no such image
+	//     schema:
+	//       "$ref": "#/types/ErrorModel"
+	//   '409':
+	//     description: conflict
+	//     schema:
+	//       "$ref": "#/types/ErrorModel"
+	//   '500':
+	//     description: unexpected error
+	//     schema:
+	//      "$ref": "#/types/ErrorModel"
 	r.Handle(VersionedPath("/images/{name:..*}/tag"), APIHandler(s.Context, handlers.TagImage)).Methods(http.MethodPost)
-
-	// commit has a different endpoint
+	// swagger:operation POST /commit/ commit commitContainer
```

Comment:
tag should be images not commit.

Reply:
it is actually a different endpoint. i think it should be commit